loadtest/config.json

```diff
@@ -15,32 +15,36 @@
     },
     "message_type_distribution": {
         "dex": {
-            "limit_order_percentage": "0.2",
-            "market_order_percentage": "0.8"
+            "place_order_percentage": "0.5",
+            "cancel_order_percentage": "0.5"
         },
         "staking": {
             "delegate_percentage": "0.5",
             "undelegate_percentage": "0.25",
             "begin_redelegate_percentage": "0.25"
         }
     },
-    "run_oracle": false,
-    "message_type": "basic",
+    "order_type_distribution": {
+        "limit_order_percentage": "0.2",
+        "market_order_percentage": "0.8"
+    },
+    "message_type": "dex",
+    "run_oracle": true,
```

Comment:
nit: you might want to leave `run_oracle` false since idt your ansible script (or the one committed in master) has the steps for getting validator keys

Reply:
Noted for next PR: https://github.com/sei-protocol/sei-chain/pull/360#discussion_r1011800396